checklist.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+permalink: /checklists/training/
+title: Checklist
+---
+## Scheduling the Event
+
+1.  Decide if it will be in person, online for one site, or online for several.
+1.  Talk through expectations with the host(s).
+    *   If it is in person, make sure the host knows they're covering travel costs.
+    *   Determine who is allowed to attend.
+1.  Arrange trainers.
+1.  Arrange space.
+    *   Make sure there are breakout rooms for video recording.
+1.  Choose dates.
+    *   If it is in person, book travel.
+1.  Get names and email addresses of attendees from host(s).
+    *   Register those people in AMY.
+
+## Setting Up
+
+1.  Set up a one-page website for the workshop.
+1.  Send the URL to the admins.
+1.  Check whether any attendees have special needs.
+1.  If it is online:
+    *   Test the video conference link.
+    *   Make sure attendees will all have network access.
+1.  Create an Etherpad.
+1.  Email attendees:
+    *   a link to the workshop home page
+    *   background readings
+
+## During the Event
+
+1.  Remind everyone of the code of conduct.
+1.  Collect attendance.
+1.  Distribute sticky notes.
+
+## After the Event
+
+1.  Email attendees about the final checkout process.
+    *   Choose one of the core lessons (or one from each Carpentry if qualifying for both).
+    *   Submit an exercise as a pull request (Software Carpentry) or to a Google Doc (Data Carpentry).
```

Comment:
Collect participants' GH handles (as per https://github.com/swcarpentry/instructor-training/issues/63) unless they are only interested in teaching DC and don't use GH.